manager/controllers/utils.go

```diff
@@ -246,9 +246,11 @@ func ApplyTagsFilter(c *gin.Context, tx *gorm.DB, systemIDExpr string) (*gorm.DB
 	tags := c.QueryArray("tags")
 	for _, t := range tags {
 		matches := tagRegex.FindStringSubmatch(t)
-		tagJSON := fmt.Sprintf(`[{"namespace":"%s", "key": "%s", "value": "%s"}]`, matches[1], matches[2], matches[3])
-		subq = subq.Where(" h.tags @> ?::jsonb", tagJSON)
-		applied = true
+		if len(matches) == 4 {
```

Comment:
would be good to add test to ensure that we handle correctly previously failing state